modules/distr-tracing-tempo-object-storage-setup-gcp-sts-install.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+//
+//* observability/distr_tracing/distr-tracing-tempo-installing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="setting-up-google-cloud-storage-with-security-token-service_{context}"]
+= Setting up the Google Cloud storage with the Security Token Service
+
+You can set up the Google Cloud Storage (GCS) with the Security Token Service (STS) by using the Google Cloud CLI.
+
+:FeatureName: The Amazon GCS storage with the Security Token Service
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+.Prerequisites
+
+* You have installed the latest version of the Google Cloud CLI.
+
+.Procedure
+
+. Create a GCS bucket on the Google Cloud Platform (GCP).
+
+. Create or reuse a service account with Google's Identity and Access Management (IAM):
++
+[source,bash]
+----
+SERVICE_ACCOUNT_EMAIL=$(gcloud iam service-accounts create <iam_service_account_name> \ # <1>
+    --display-name="Tempo Account" \
+    --project <project_id>  \ # <2>
+    --format='value(email)' \ 
+    --quiet)
+----
+<1> The name of the service account on the GCP.
+<2> The project ID of the service account on the GCP.
+
+. Bind the required GCP roles to the created service account at the project level:
++
+[source,terminal]
+----
+$ gcloud projects add-iam-policy-binding <project_id> \
+    --member "serviceAccount:$SERVICE_ACCOUNT_EMAIL" \
+  --role="roles/iam.workloadIdentityUser" \
+&&
+  gcloud projects add-iam-policy-binding <project_id> \
+    --member "serviceAccount:$SERVICE_ACCOUNT_EMAIL" \
+    --role "roles/storage.objectAdmin"
+----
+
+. Retrieve the `POOL_ID` value of the Google Cloud Workload Identity Pool that is associated with the cluster:
```

Comment:
This step might differ in customer env. We need to mention that the command is only for reference.